cluster/gce/gci/configure-helper.sh

```diff
@@ -1858,6 +1858,17 @@ function prepare-etcd-manifest {
   fi
   # Replace the volume host path.
   sed -i -e "s@/mnt/master-pd/var/etcd@/mnt/disks/master-pd/var/etcd@g" "${temp_file}"
+  # Replace the runAsUser and runAsGroup
+  if [[ -n "${ETCD_RUNASUSER:-}" && -n "${ETCD_RUNASGROUP:-}" ]]; then
+    sed -i -e "s@{{runAsUser}}@${ETCD_RUNASUSER}@g" "${temp_file}"
+    sed -i -e "s@{{runAsGroup}}@${ETCD_RUNASGROUP}@g" "${temp_file}"
+    chown -R ${ETCD_RUNASUSER}:${ETCD_RUNASGROUP} /var/etcd/
+    chown -R ${ETCD_RUNASUSER}:${ETCD_RUNASGROUP} /var/log/etcd.log
```

Comment:
This is already done on line 1893 and 1894

Reply:
Fixed